tests/python/test_python_frontend.py

```diff
@@ -126,6 +82,12 @@ def inner_fn(*args, **kwargs):
     return inner_fn
 
 
+def setUpModule():
+    from utils import atexit_serde_check
+
+    atexit_serde_check()
+
```

Comment:
Can we use this as a decorator directly?

Reply:
`atexit_serde_check` isn't a decorator. It is a regular function. `atexit_serde_create_op_test` is a pseudo-decorator because it creates many functions from a single function.

Reply:
Right, sorry, I misread.
A follow-up question then is: Can we use `atexit_serde_check` directly? Why is the wrapper around it necessary?

Reply:
`setUpModule` is a unittest fixture that runs once before running any tests in `TestNvFuserFrontend`. We only want to call `atexit_serde_check` once.